test/parallel/test-net-connect-local-error.js

```diff
@@ -3,25 +3,46 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+const fixedPort = 8080;
```

Comment:
@sebastianplesciuc I think they convinced me to move the test to `/sequential/` there it's Ok.